docs/src/main/sphinx/release/release-392.md

```diff
@@ -0,0 +1,70 @@
+# Release 392 (2 August 2022)
+
+## General
+
+* Add support for dynamic filtering with task retries. ({issue}`9935`)
+* Add support for correlated sub-queries for ``DELETE`` operations. ({issue}`9447`) 
+* Fix potential query failure in certain complex queries with multiple joins and
+  aggregations. ({issue}`13315`)
+
+## JDBC driver
+
+* Add the ``assumeLiteralUnderscoreInMetadataCallsForNonConformingClients``
+  configuration property as a replacement for
+  ``assumeLiteralNamesInMetadataCallsForNonConformingClients``, which is
+  deprecated and planned to be removed in a future release. ({issue}`12761`)
+* Add report for total time elapsed reading data from JDBC sources. ({issue}`13132`)
```

Comment:
there is a report in the JDBC driver? sounds weird

Reply:
Other option was to include it on all the other JDBC connectors, I'll move it there.

Reply:
The current sections is wrong. Please move to JDBC **_connectors_**.

Reply:
It should be copied and be in each specific connector.. we abandoned the JDBC connectors section usage since its is not clear to the user what these are .. nor is it always consistent

Reply:
nit: Language feels weird. Would `Report the total time spent reading data from the source` (in EXPLAIN ANALYZE VERBOSE, the web UI and event listeners)?

The part in brackets I consider to be too much information but others may think differently.